Jellyfin.Data/Jellyfin.Data.csproj

```diff
@@ -4,7 +4,6 @@
     <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
```suggestion
    <GenerateDocumentationFile>true</GenerateDocumentationFile>
    <TreatWarningsAsErrors Condition=" '$(Configuration)' == 'Release' ">true</TreatWarningsAsErrors>
    <NoWarn Condition=" '$(Configuration)' == 'Release' ">1591,1573</NoWarn>
```

Reply:
entries in NoWarn are semicolon delimited, afaik

Reply:
Just tested, both `;` and `,` work so 🤷

Reply:
This misses the point of all the pragma warnings, that new files need to be fully documented.